tox.ini

```diff
@@ -67,7 +67,8 @@ commands =
       --skip {toxinidir}/lib \
       --skip {toxinidir}/venv \
       --skip {toxinidir}/.mypy_cache \
-      --skip {toxinidir}/tests/integration/.mypy_cache
+      --skip {toxinidir}/tests/integration/.mypy_cache \
+      --skip {toxinidir}/tests/integration/flask_test_app/.mypy_cache
```

Comment:
What is the reason for skipping this please?

Reply:
This is a left-over of me trying to put it in the `test/integration` directory

Reply:
So I guess we can remove it?

Reply:
Removed